pkg/controller/controller.go

```diff
@@ -513,7 +513,7 @@ func makeAnnotations(fni *faasv1.FunctionIngress) map[string]string {
 			annotations["zalando.org/skipper-filter"] = `setPath("/function/` + fni.Spec.Function + `")`
 			break
 		case "traefik":
-			annotations["traefik.ingress.kubernetes.io/rewrite-target"] = "/function/" + fni.Spec.Function + "/$1"
```

Comment:
Why would you say that $1 is no longer required?

Reply:
Because traefik internally converts the annotations to the following routing rule: `PathPrefix:/;ReplacePathRegex: ^/(.*) /function/podinfo/$1$1`
Note the two replacements (`$1`). So all request will be forwarded to the function with incorrect path (older versions of Traefik, except for the root path) or won't be routed at all because of indefinite redirects (latest versions of Traefik).